src/cluster.c

```diff
@@ -4410,6 +4410,43 @@ int getSlotOrReply(client *c, robj *o) {
     return (int) slot;
 }
 
+int getSlotStatus(client *c, unsigned char *slots, int del, int start_slot, int end_slot) {
+    int slot;
+    for(slot = start_slot; slot <= end_slot; slot++){
+        if (del && server.cluster->slots[slot] == NULL) {
+            addReplyErrorFormat(c,"Slot %d is already unassigned", slot);
+            return C_ERR;
+        } else if (!del && server.cluster->slots[slot]) {
```

Comment:
else is not needed here because of the return statement on the previous line

Reply:
Will Update, Thanks